guides/shop-managers-guide/41-accounts-management-and-invite.md

```diff
@@ -0,0 +1,28 @@
+# Accounts
```

Comment:
Why does the numbering of these jump up to 41? Are there 40 preceding documents?

Reply:
Also where are there still empty files?

Reply:
My mistake for leaving it out but can you add a doc for configuring address validation?

Reply:
Files numbered 40s is the 4th item in Admin side bar (Accounts). The jump in numbers is so that I can add more files if needed without changing the numbers. 
Maybe the blank file is from trunk. I will add address validation to it.